libs/utils/utils.js

```diff
@@ -648,7 +694,9 @@ export async function loadArea(area = document) {
     await decorateIcons(section.el, config);
 
     // Post LCP operations.
-    if (isDoc && section.el.dataset.idx === '0') { loadPostLCP(config); }
+    if (isDoc && section.el.dataset.idx === '0') {
```

Comment:
Same as previous comment. Change back. Lines matter.

The ideal size of utils is ~600 lines (which we're not close to any more)